libraries/api.rb

```diff
@@ -0,0 +1,33 @@
+module Nexus3
+  class Api
+    require 'json'
+
+    def initialize(url, user, password)
+      @http_client = HTTPClient.new(url).tap do |client|
```

Comment:
you should pass your url using the `base_url` name parameter (https://github.com/nahi/httpclient/blob/master/lib/httpclient.rb#L398)